index.html

```diff
@@ -598,14 +598,41 @@ <h3>Summary</h3>
 
     <section id="conformance">
 
-      <p>
-      </p>
      <p>
-A conforming <dfn>VCALM client</dfn> is ...
+A conforming <dfn>issuer service implementation</dfn> MUST provide the interface
+described in Section [[[#issue-credential]]]. Other interfaces described in
+Section [[[#issuing]]] MAY also be provided.
+     </p>
+
+     <p>
+A conforming <dfn>verifier service implementation</dfn> MUST provide the
+interface described in Section [[[#verify-credential]]] and Section
+[[[#verify-presentation]]]. Other interfaces described in Section
+[[[#verifying]]] MAY also be provided.
+     </p>
+
+     <p>
+A conforming <dfn>holder service implementation</dfn> MUST provide the interface
+described in Section [[[#get-exchange-protocols]]], Section
+[[[#participate-in-an-exchange]]], and Section [[[#create-presentation]]].
```

Comment:
I wonder if creating a presentation via the VC API should be strictly necessary for holder conformance, or if simply implementing exchange participation would be sufficient (whereby presentation creation may all be done locally).

Reply:
Fixed.